src/extractor.coffee

```diff
@@ -106,6 +106,34 @@ module.exports =
 
     null
 
+  # Find any links in the doc
+  links: (doc, topNode, lang) ->
+    gravityItems = []
+    links = []
+    removeGravityItems = (gravityItems) ->
```

Comment:
Why did you need to remove these nodes to find the links? Just curious what the logic is here. Does removing these items cause any side effects to the text extraction later?